site/en/docs/privacy-sandbox/attribution-reporting/system-overview/index.md

```diff
@@ -127,17 +127,15 @@ following kinds of information:
 
 *  Contextual reporting data, such as the ad creative ID, information about the
    campaign, or geography.
-*  A conversion target, as in the site that is the target of the ad where you
-   hope the user will end. This tells the browser what source events correspond
-   to specific conversions.
+*  A conversion destination, as in the site where you expect the user will convert.
```

Comment:
So, I'm just looking at this in github so I have no context but maybe adding an object will help this sentence? Convert "what"?

Reply:
These are Maud's edits. It could say " conversion destination, as in the site where you expect the conversion to take place" but that's a bit redundant?